warpcast.json

```diff
@@ -886,5 +886,11 @@
         "parent_url": "https://rarible.com",
         "image": "https://warpcast.com/~/channel-images/rarible.jpg",
         "channel_id": "rarible"
+     },
+    {
```

Comment:
Hey @vp1337 I'll let @manan19 take this from here, but the content looks indeed correct.

Would you mind tidying up the JSON, it looks unformatted. Thanks!

Reply:
<img width="146" alt="image" src="https://github.com/neynarxyz/farcaster-channels/assets/847179/5f546af8-fdc8-4818-9c63-f01dd00fde90">

Reply:
I edited the misalignment in Codespaces. Does it automatically edit the PR or should make additional changes?

Reply:
Unfortunately the brackets do not look aligned in this code :(

Reply:
Should be good now, excuse me for the spam.

Reply:
It's perfect